mods.md

```diff
@@ -22,8 +22,8 @@ Galacticraft Planets  | 3.0.12               |
 Guilded Games Utility | 1.7.10-1.1           | <http://ae-mod.info/>
 Hats                  | 4.0.1                | <http://ichun.us/mods/hats/>
 iChunUtil             | 4.2.3                | <http://www.ichun.us/>
-Iguana Tinker Tweaks  | 2.1.6                |
-Inventory Tweaks      | 1.59-dev-152-cf6e263 |
+Iguana Tinker Tweaks  | 2.1.6                | <https://www.curseforge.com/minecraft/mc-mods/iguanas-tinker-tweaks>
```

Comment:
Can you use this link instead? `https://minecraft.curseforge.com/projects/iguanas-tinker-tweaks?gameCategorySlug=mc-mods&projectID=223248`
Pretty sure I specified only http://minecraft.curseforge.com/* are allowed :)